### ISSUE TITLE
Nested `expose` support

### ISSUE THREAD

**Issue by mxmCherry** [ruby-grape/grape-swagger-entity#18]
This code:

```ruby
class Entity < Grape::Entity
  expose :object do
    expose :prop, documentation: { desc: 'Prop' } { |entity, _| ... }
  end
end
```

Results in:

```
{
    ...
    "definitions": {
        "Entity": {
            "type": "object",
            "properties": {
                "object": {
                    "type": "string"
                }
            }
        }
    }
}
```

So shown in swagger UI as:

```
Entity {
  object (string, optional)
}
```

When using another entity for `object` prop (`expose :object, using: ObjectEntity`) doc is generated correctly.

Relevant Gemfile part:

```
gem 'grape', '0.19.1'
gem 'grape-entity', '0.6.0'
gem 'grape-swagger', '0.26.0'
gem 'grape-swagger-entity', '0.1.5'
```

**Comment by Bugagazavr** [comment]
Hello, sorry for delay i try to look this

**Comment by Bugagazavr** [comment]
Resolved in 0.2.0 release
